pkgs/development/python-modules/whool/default.nix

```
@@ -0,0 +1,55 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, git
+, hatch-vcs
+, importlib-metadata
+, lib
+, manifestoo-core
+, pytestCheckHook
+, pythonOlder
+, tomli
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "whool";
+  version = "1.0.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "sbidoul";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-skJoMDIgZgRjfp4tsc6TKYVe09XBvg8Fk2BQfqneCYI=";
+  };
+
+  build-system = [ hatch-vcs ];
+
+  dependencies = [
+    manifestoo-core
+    wheel
+  ] ++ lib.optionals (pythonOlder "3.11") [
+    tomli
+  ] ++ lib.optionals (pythonOlder "3.8") [
+    importlib-metadata
+  ];
+
+  pythonImportsCheck = [ "whool" ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    git
+  ];
+
+  setupHook = ./setup-hook.sh;
+
+  meta = with lib; {
+    description = "Standards-compliant Python build backend to package Odoo addons";
+    homepage = "https://github.com/acsone/whool";
+    changelog = "https://github.com/acsone/whool/blob/${version}/CHANGELOG.md";
```

```suggestion
    homepage = "https://github.com/sbidoul/whool";
    changelog = "https://github.com/sbidoul/whool/blob/${version}/CHANGELOG.md";
```

This page doesn't exist